api/consumer/record-locator/consumer.yaml

```diff
@@ -122,7 +122,22 @@ info:
 
     There are [libraries and SDKs available](https://digital.nhs.uk/developer/guides-and-documentation/api-technologies-at-nhs-digital#fhir-libraries-and-sdks) to help with FHIR API integration.
 
+    ### Open source
+
+    You might find the following open source resources useful:
+
+    | Resource                  | Description                                               | Links                                                                      |
+    |---------------------------|-----------------------------------------------------------|----------------------------------------------------------------------------|
+    | NRL v3 API                | Source code for the core API and sandbox                  | [GitHub repo](https://github.com/NHSDigital/nrlf)                          |
+    | NRL v3 Producer API       | Source code for the producer API proxy and specification. | [GitHub repo](https://github.com/NHSDigital/nrl-producer-api)              |
+    | NRL v3 Consumer API       | Source code for the consumer API proxy and specification. | [GitHub repo](https://github.com/NHSDigital/nrl-consumer-api)              |
```

Comment:
This is missing lambda packaging and lambda logging here and the errors are still below the open source heading.